### DISCUSSION TITLE
Using conv_only in Convolutional layers

### DISCUSSION THREAD

**Discussion by Can-Zhao** [Project-MONAI/GenerativeModels#396] [General]
Hi,

I found that `conv_only=True` is almost always used in MONAI Generative Models, like https://github.com/Project-MONAI/GenerativeModels/blob/b1c1a459c98a4d55570fe775fdcfad1707dae71b/generative/networks/nets/autoencoderkl.py#L137
While in official Stable Diffusion code, `bias=True` is used as this is the default in [torch.nn.Conv2d](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html) like in [stable diffusion v1 ](https://github.com/CompVis/stable-diffusion/blob/21f890f9da3cfbeaba8e2ac3c425ee9e998d5229/ldm/modules/diffusionmodules/model.py#L92) and [stable diffusion v2](https://github.com/Stability-AI/stablediffusion/blob/cf1d67a6fd5ea1aa600c4df58e5b47da45f6bdbf/ldm/modules/diffusionmodules/model.py#L100).

I was wondering if this modification was expected?

Thank you!!

**Comment by Can-Zhao** [comment]
@Warvito @ericspod What do you think? Thank you!

**Comment by ericspod** [comment]
The `conv_only=True` argument means that the block will not have activation/regularisation/dropout layers, ie. just the convolutional layer, but this will still have the bias term if the `bias` argument is not changed from its [default of True](https://github.com/Project-MONAI/GenerativeModels/blob/5657b8f2d297e2d2b09017cf9f57dad2a72186f4/monai/networks/blocks/convolutions.py#L112).

**Reply by Can-Zhao** [reply]
Got it! Thank you for the explanation!
